src/main/kotlin/io/snyk/plugin/settings/SnykProjectSettingsConfigurable.kt

```diff
@@ -88,6 +92,13 @@ class SnykProjectSettingsConfigurable(val project: Project) : SearchableConfigur
             getSyncPublisher(project, SnykResultsFilteringListener.SNYK_FILTERING_TOPIC)?.filtersChanged()
             getSyncPublisher(project, SnykProductsOrSeverityListener.SNYK_ENABLEMENT_TOPIC)?.enablementChanged()
         }
+
+        runBackgroundableTask("Identifying with Analytics service", project, true) {
```

Comment:
Why was it moved here?

Reply:
It was blocking the UI thread, which is bad - on my computer, Sentry & Amplitude are DNS blocked, so the connection times out only after 60s and would block the UI for that amount of time.